PowerEditor/src/localizationString.h

```diff
@@ -115,5 +115,6 @@ LocalizationSwitcher::LocalizationDefinition localizationDefs[] =
 	{TEXT("Corsu"), TEXT("corsican.xml")},
 	{TEXT("Brezhoneg"), TEXT("breton.xml")},
 	{TEXT("کوردی‬"), TEXT("kurdish.xml")},
-	{TEXT("Pig latin"), TEXT("piglatin.xml")}
-};
\ No newline at end of file
+	{TEXT("Pig latin"), TEXT("piglatin.xml")},
+        {TEXT("Zulu"), TEXT("zulu.xml")}

```

Comment:
Spaces instead of Tab

Reply:
Check this already done https://prnt.sc/mbv07w

Reply:
@chcg this is the link https://notepad-plus-plus.org/contribute/binary-translations.html where not display Zulu language yet it's the Russian language